src/sagemaker/huggingface/training_compiler/config.py

```diff
@@ -24,7 +26,7 @@
 class TrainingCompilerConfig(BaseConfig):
     """The SageMaker Training Compiler configuration class."""
 
-    SUPPORTED_INSTANCE_CLASS_PREFIXES = ["p3", "g4dn", "p4"]
+    SUPPORTED_INSTANCE_CLASS_PREFIXES = ["p3", "g4dn", "p4d", "g5"]
```

Comment:
did you mean to remove p4? is it intentional?

Reply:
`p4d` is the only available flavor of `p4` AFAIK. Just making the check more specific.